srp_daemon/srp_ib_types.h

```diff
@@ -344,8 +326,8 @@ typedef struct _ib_mad_notice_attr		// Total Size calc  Accumulated
 			uint8_t		sl;		// 1
 			__be32		qp1;		// 4
 			__be32		qp2;		// 4
-			ib_gid_t	gid1;		// 16
-			ib_gid_t	gid2;		// 16
+			union ibv_gid	gid1;		// 16
+			union ibv_gid	gid2;		// 16
 		} PACK_SUFFIX ntc_257_258;
```

Comment:
This layout prior to change does not match IB spec for this trap. Is it used by SRP ?

In the spec, I don't think the GIDs are 8 byte aligned. I think they're 4 byte aligned.